src/pages/docs/[[...slug]]/index.tsx

```diff
@@ -25,18 +52,36 @@ const DocsDetailPage = ({ doc, prev, next }: DocsDetailPageProps) => {
 
       <LayoutHeader />
 
-      <LayoutContainer className="flex min-h-[calc(100dvh-var(--sz-64))] w-full gap-2xl">
+      <LayoutContainer className="flex min-h-[calc(100dvh-var(--sz-64))] w-full gap-lg">
         <LayoutSideNav />
 
-        <main className="flex w-full flex-row gap-2xl">
-          <div className="relative flex min-w-0 flex-1 flex-col">
-            <MDXComponent
-              code={doc.body.code}
-              globals={{ examples: doc.examples, docgen: doc.docgen }}
+        <main className="flex w-full flex-row">
+          <article className="flex w-full flex-col">
+            <DocHeader
+              {...{
+                title,
+                version,
+                name,
+                category,
+                license,
+                description,
+                keywords,
+                packageUrl,
+                bugReportUrl,
+              }}
+              className="-mx-lg px-lg pb-lg md:mx-none"
             />
-            <DocFooter previous={prev} next={next} filePath={doc._raw.sourceFilePath} />
-          </div>
-          <DocsTableOfContent headings={doc.headings} />
+            <div className="flex w-full grow flex-row gap-lg">
+              <div className="relative flex min-w-0 flex-1 flex-col md:pl-lg">
+                <MDXComponent
+                  code={doc.body.code}
+                  globals={{ examples: doc.examples, docgen: doc.docgen }}
+                />
+                <DocFooter previous={prev} next={next} docUrl={bugReportUrl} />
```

Comment:
Same as the the header I think that would be better to place this component inside `Docs` folder with `Docs` preffix